readme.txt

```diff
@@ -40,6 +40,10 @@ This section describes how to install the plugin and get it working.
 
 == Frequently Asked Questions ==
 
+= Why is a Jetpack Connection and WordPress.com account required? =
+
+We use the Jetpack connection to authenticate each request and, if you use the shipping label service, to charge your credit card on file.
+
```

Comment:
To re-order the list in https://wordpress.org/plugins/woocommerce-services/#faq.